devel/README.md

```diff
@@ -228,7 +228,7 @@ reviews much easier.
 Using Mixer as an example:
 
 ```shell
-cd $(ISTIO)/mixer
+cd $ISTIO/mixer
```

Comment:
oh right it should have been ${ISTIO} or $ISTIO indeed ($(ISTIO) is in Makefiles)

thanks for having fixed it